backend/src/apiserver/server/api_converter.go

```diff
@@ -148,14 +152,45 @@ func toApiParameters(paramsString string) ([]*api.Parameter, error) {
 	return apiParams, nil
 }
 
+func toApiRuntimeParameters(paramsString string) (map[string]*structpb.Value, error) {
+	if paramsString == "" {
+		return make(map[string]*structpb.Value), nil
+	}
+	runtimeParamsStringsMap := make(map[string]string)
+	err := json.Unmarshal([]byte(paramsString), &runtimeParamsStringsMap)
+	if err != nil {
+		return nil, util.NewInternalServerError(err, "Cannot unmarshal RuntimeConfig Parameter to map[string]string")
+	}
+	runtimeParams := make(map[string]*structpb.Value)
+	for k, v := range runtimeParamsStringsMap {
+		protoValue := structpb.NewStringValue("")
```

Comment:
why `NewStringValue`?

Reply:
Just to initialize the variable to be later marshalled into.